src/robusta/integrations/google_chat/sender.py

```diff
@@ -0,0 +1,141 @@
+from robusta.core.reporting.blocks import (
+    LinksBlock,
+    LinkProp,
+    MarkdownBlock, FileBlock, HeaderBlock, TableBlock, ListBlock,
+)
+from robusta.core.reporting.consts import FindingSource
+from robusta.core.sinks.google_chat.google_chat_params import GoogleChatSinkParams
+
+import requests
+
+
+class GoogleChatSender:
+    def __init__(self, params: GoogleChatSinkParams, signing_key, account_id, cluster_name):
+        self.params = params
+        self.signing_key = signing_key
+        self.account_id = account_id
+        self.cluster_name = cluster_name
+
+    def send_finding(self, finding: Finding, platform_enabled: bool):
+        blocks: List[BaseBlock] = []
+
+        status: FindingStatus = (
+            FindingStatus.RESOLVED if finding.title.startswith("[RESOLVED]") else FindingStatus.FIRING
+        )
+        blocks.append(self.__create_finding_header(finding, status))
+
+        if platform_enabled:
+            blocks.append(self.__create_links(finding))
+
+        blocks.append(MarkdownBlock(text=f"*Source:* `{self.cluster_name}`\n"))
+        if finding.description:
+            if finding.source == FindingSource.PROMETHEUS:
+                blocks.append(MarkdownBlock(f"{Emojis.Alert.value} *Alert:* {finding.description}"))
+            elif finding.source == FindingSource.KUBERNETES_API_SERVER:
+                blocks.append(
+                    MarkdownBlock(f"{Emojis.K8Notification.value} *K8s event detected:* {finding.description}")
+                )
+            else:
+                blocks.append(MarkdownBlock(f"{Emojis.K8Notification.value} *Notification:* {finding.description}"))
+
+        for enrichment in finding.enrichments:
+            items = [
+                self.__block_to_markdown_text(block)
+                for block in enrichment.blocks
+                if not isinstance(block, FileBlock)
+            ]
+            if items:
+                blocks.extend(items)
+
+        data = self.__blocks_to_data(blocks)
+        self.__send_to_api(data)
+
+    def __blocks_to_data(self, blocks: List[BaseBlock]) -> Dict:
+        text = ""
+        for block in blocks:
+            appendix = self.__block_to_markdown_text(block)
+            if appendix is not None:
+                text += appendix
+        return {"text": text}
+
+    def __block_to_markdown_text(self, block: BaseBlock) -> str:
+        if isinstance(block, MarkdownBlock):
+            return block.text
+        elif isinstance(block, FileBlock):
+            # We ignore this case as we are not able to send attachments
+            # using the Google Chat webhook API.
+            return None
+        elif isinstance(block, HeaderBlock):
+            return MarkdownBlock(block.text).text
+        elif isinstance(block, LinksBlock):
+            return self.__format_links(block.links) + "\n\n"
+        elif isinstance(block, TableBlock):
+            if len(block.headers) == 2:
```

Comment:
Can we use tabulate to convert the table to string?
see `to_table_string` in `blocks.py`

Reply:
Additional note: tables wider than 2 columns are rendered using `to_table_string`